content/pages/volunteers.md

```diff
@@ -71,17 +76,6 @@ Here are the volunteer positions currently available.
   positions. In addition to attending the event, you will also have to spend
   time familiarizing yourself with the material prior to it.
 
-### Systems Committee Volunteer
```

Comment:
I think this can stay up on the site, just make it clear we aren't accepting people for the roles.